### PR TITLE
change rendering from travis to github actions

### PR DESCRIPTION
**Pull request by tbugfinder**

### CODE REVIEW

File: .github/workflows/push.sh [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  git branch temp-changes
+  git checkout $GITHUB_REF_SLUG
+  git merge --no-commit --no-ff temp-changes
+  git reset HEAD -- package.json
+  git reset HEAD -- package-lock.json
+  git commit --message "GitHub Actions Build: $GITHUB_RUN_ID [ci skip]"
+}
+
+upload_files() {
+  git remote add origin-pages https://${GH_TOKEN}@github.com/${GITHUB_REPOSITORY}.git > /dev/null 2>&1
```
Comment (michaelsauter) [inline .github/workflows/push.sh:22]: Do you know if `GH_TOKEN` is available? Or is that a Travis specific thing?

Comment (tbugfinder) [reply .github/workflows/push.sh:22]: `GH_TOKEN` must be pre-defined. It is an ENV setting in travis.
For my tests I've defined it as a GitHub project secret.

https://github.com/opendevstack/ods-documentation/pull/62/files#diff-edd7e4675f0f52ec8d2f51cb50523e3e4cecfcf42b94f3b314d15c0a5d283d2cR28

Comment (michaelsauter) [reply .github/workflows/push.sh:22]: Ah okay cool. Did you look into `GITHUB_TOKEN`, as described in https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow? Just curious, happy to merge this as is and improve later ...

Comment (tbugfinder) [reply .github/workflows/push.sh]: I wasn't aware of GITHUB_TOKEN, but changed to it as actions are skipped on push (built-in).

**Review by michaelsauter — APPROVED** [review]
Awesome, let's try it! :)

**Comment by tbugfinder** [comment]
I haven't had permission to merge, however it looks like travis-ci.org doesn't run any longer :-)
Are you still ok with squash&merge?

**Comment by michaelsauter** [comment]
@tbugfinder done. I'm confused thought why you don't have the permissions, you are part of the contributors team which has write permissions ...

**Pull request merged**